lnrpc/routerrpc/router.proto

```diff
@@ -367,6 +367,25 @@ message PairData {
     bool last_attempt_successful = 3 [json_name = "last_attempt_successful"];
 }
 
+message QueryMcProbabilityRequest{
+    /// The source node pubkey of the pair.
+    bytes node_from = 1 [json_name = "node_from"];
+
+    /// The destination node pubkey of the pair.
+    bytes node_to = 2 [json_name = "node_to"];
+
+    /// The amount for which to calculate a probability.
+    int64 amt_msat = 3 [json_name = "amt_msat"];
+}
+
+message QueryMcProbabilityResponse{
+    /// The success probability for the requested pair.
+    double probability = 1 [json_name = "probability"];
+
+    /// The historical data for the requested pair.
+    PairData history = 2 [json_name = "history"];
```

Comment:
is the reason for `PairData` to reuse it here?

Reply:
yes